charts/newrelic-infrastructure/values.yaml

```diff
@@ -185,6 +185,11 @@ openshift:
 # Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
 # priorityClassName: high-priority
 
+# Allows to add additional environment variables
+# extraEnv: 
```

Comment:
could you follow the same approach in the values https://github.com/newrelic/nri-kubernetes/blob/e1cbd0007575f47296afe88534fc924c5e9bdd60/charts/newrelic-infrastructure/values.yaml#L99 ?

Reply:
Hello! Updated PR, please review

Reply:
Updated the PR, please review